docs/rules/forbid-dom-props.md

```diff
@@ -36,14 +36,17 @@ The following patterns are **not** considered warnings:
 
 ```js
 ...
-"react/forbid-dom-props": [<enabled>, { "forbid": [<string>] }]
+"react/forbid-dom-props": [<enabled>, { "forbid": [<string>], "message": <string> }]
```

Comment:
similarly here - it seems like instead of an overarching message property, it'd be better to allow each forbid value to be a string *or* an object, with the object form allowing a custom message?